scripts/cabal-format.sh

```diff
@@ -2,6 +2,23 @@
 
 set -euo pipefail
 
-for f in $(git ls-files -- '*.cabal'); do cabal format $f; done
+if [[ "${1:-format}" != "format" && "${1:-format}" != "check" ]]; then
+  echo "Error: Invalid mode. Allowed values are 'format' or 'check'." >&2
```

Comment:
Let's report the supplied mode, which will be useful realizing the wrong argument.
```suggestion
  echo "Error: Invalid mode: ${mode}. Allowed values are 'format' or 'check'." >&2
```